src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilsTest.java

```diff
@@ -63,7 +67,7 @@ public void testGetIntFromInaccessibleField() throws NoSuchFieldException {
 
     @Test
     public void testTokenValueIncorrect() {
-        final Integer id = Integer.MAX_VALUE - 1;
+        final Integer id = 184;
```

Comment:
One of pitest's mutations is to change "<" to "<=" and vice versa. In case of `Integer.MAX_VALUE-1` it didn't change anything and mutation survived.